cfme/intelligence/chargeback/assignments.py

```diff
@@ -113,7 +107,15 @@ def _fill(self, view):
         return view.fill(fill_details)
 
 
-@navigator.register(Assign, 'All')
+@attr.s
+class AssignCollection(BaseCollection):
```

Comment:
**REQUIRED**
Please use a plural name for collections: `class AssingsCollection`

Reply:
Sure, Wasn't aware of the convention but I'll keep that in mind :)